substrate-examples/offchain-worker/README.md

```diff
@@ -69,11 +69,48 @@ the warning is not generated.
 
 ## Open issues
```

Comment:
Can you add a line, that all the tests were done on the `devvirtualbox`. Then you can leave it out below (except for the one, where we tested both). (And can you also add the line in the pallet-template for me?)